aws-auth-cognito/build.gradle.kts

```diff
@@ -38,7 +38,7 @@ dependencies {
     testImplementation(project(":testutils"))
     testImplementation(project(":aws-auth-cognito"))
     //noinspection GradleDependency
-    testImplementation("org.json:json:20180813")
+    testImplementation("org.json:json:20210307")
```

Comment:
Let's move these dependency declarations to our version catalog in `settings.gradle`